aspnet/mvc/controllers/dependency-injection.rst

```diff
@@ -118,4 +118,4 @@ Once you've specified a strongly-typed configuration object (in this case, ``Sam
   :emphasize-lines: 3,5,7
   :lines: 7-22
 
-Following the Options pattern allows settings and configuration to be decoupled from one another, and ensures the controller is following `separation of concerns <http://deviq.com/separation-of-concerns/>`_, since it doesn't need to know how or where to find the settings information. It also makes the controller easier to :doc:`unit test </testing/unit-testing>`, since there is no `static cling <http://deviq.com/static-cling/>`_ or direct instantiation of settings classes within the controller class.
+Following the Options pattern allows settings and configuration to be decoupled from one another, and ensures the controller is following `separation of concerns <http://deviq.com/separation-of-concerns/>`_, since it doesn't need to know how or where to find the settings information. It also makes the controller easier to `unit test <https://docs.microsoft.com/en-us/dotnet/articles/core/testing/unit-testing-with-dotnet-test>`_, since there is no `static cling <http://deviq.com/static-cling/>`_ or direct instantiation of settings classes within the controller class.
```

Comment:
This one should probably also link to https://docs.asp.net/en/latest/mvc/controllers/testing.html.